CONTRIBUTING.md

```diff
@@ -583,67 +611,84 @@ class LoginForm(Form):
 # pylint: enable=too-few-public-methods
 ```
 
-# Git hooks
+# Pre-commit hooks
 
-Another great way of automating linting and testing is to use
- [Git Hooks](https://git-scm.com/book/uz/v2/Customizing-Git-Git-Hooks). For example you could create a
-`pre-commit` file based on the Travis CI Pipeline so that before each commit a local pipeline will be
-triggered and if this pipeline fails (returns an exit code other than `0`) the commit does not come through.
-This "in theory" has the advantage that you can not commit any code that fails that again reduces the
-errors in the Travis CI Pipelines.
+Pre-commit hooks are fantastic way of speeding up your local development cycle - you can run exactly the same
+static code checks as are run in TravisCI but only limit the checks to the files you are just committing.
```

Comment:
code checks that are running in TravisCI while limiting the checks to the files you are committing.*

Reply:
Reworded.